WalletWasabi.Fluent/Infrastructure/ClipboardObserver.cs

```diff
@@ -11,16 +11,21 @@ namespace WalletWasabi.Fluent.Infrastructure;
 
 internal class ClipboardObserver
 {
+	private readonly IObservable<string?> _textChanged;
+
 	public ClipboardObserver(WalletBalances walletBalances)
 	{
 		WalletBalances = walletBalances;
+		_textChanged = ApplicationHelper.ClipboardTextChanged(RxApp.MainThreadScheduler)
+			.Replay()
+			.RefCount();
```

Comment:
Was it because of memory management in .NET?

Reply:
I'm almost sure that wasn't the reason.
I'm still unsure of what happened 100%. My impression is that the subscription was made **after** the relevant data had been pushed already. That, and that null values were being pushed instead consistent clipboard data. 

I still need to take a look to know the exact reason 😅